src/theme.ts

```diff
@@ -278,6 +278,7 @@ export enum ColorSetting {
   System = 'system',
   Dark = 'dark',
   Light = 'light',
+  Custom = 'custom',
```

Comment:
I'm not sure why these were added. Is this related to removing the Chapel Credit display?